File: two-sum/two-sum.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function(nums, target) {
    for(var i = 0; i < nums.length; i++) {
        var firstNum = nums[i];
        for(var j = i + 1; j < nums.length; j++) {
            var secNum = nums[j];
            if(firstNum + secNum === target) {
                return [i, j];
            }
        }
    }
};

File: two-sum/two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        res = []
        length = len(nums)
        for i in range(0, length):
            for j in range(i + 1, length):
                if nums[i] + nums[j] == target:
                    res.append(i)
                    res.append(j)
                    return res
                    
                    
        